Resources/Prototypes/Catalog/VendingMachines/Inventories/nutri.yml

```diff
@@ -15,7 +15,6 @@
     EZNutrientChemistryBottle: 3
     Bucket: 3
     DiseaseSwab: 20
-    #TO DO:
-    #plant analyzer
+    PlantAnalyzer: 3
```

Comment:
And if this was not available at round start, but instead researchable from science.

Reply:
Science should be used to unlock better equipment so it feels like a bonus, not basics things that one needs to do their job, pretty sure multiple changes to item research in the past weeks were going in that direction?